named.go

```diff
@@ -282,7 +282,7 @@ func bindArray(bindType int, query string, arg interface{}, m *reflectx.Mapper)
 	if arrayLen == 0 {
 		return "", []interface{}{}, fmt.Errorf("length of array is 0: %#v", arg)
 	}
-	var arglist = make([]interface{}, 0, len(names)*arrayLen)
+	arglist := make([]interface{}, 0, len(names)*arrayLen)
```

Comment:
This change was made by `gofmt` and so were a bunch of other similar changes.

Reply:
I think you're using gofumpt rather than gofmt? gofmt doesn't make these kind of changes anyway.